vector/src/main/java/im/vector/app/features/roomdirectory/createroom/CreateRoomFragment.kt

```diff
@@ -66,20 +63,11 @@ data class CreateRoomArgs(
         val openAfterCreate: Boolean = true
 ) : Parcelable
 
-<<<<<<< HEAD
-class CreateRoomFragment @Inject constructor(
-        private val createRoomController: TchapCreateRoomController,
-        private val createSpaceController: CreateSubSpaceController,
-        colorProvider: ColorProvider,
-        clock: Clock,
-) : VectorBaseFragment<FragmentCreateRoomBinding>(),
-        TchapCreateRoomController.Listener,
-=======
 @AndroidEntryPoint
 class CreateRoomFragment :
         VectorBaseFragment<FragmentCreateRoomBinding>(),
+        TchapCreateRoomController.Listener,
         CreateRoomController.Listener,
```

Comment:
We don't need this listener, TchapCreateRoomController already inherit from it

Reply:
Also, you should have this inject in the following lines:
`@Inject lateinit var createRoomController: CreateRoomController`
We should use the tchap controller:
`@Inject lateinit var createRoomController: TchapCreateRoomController`